src/lib/loadExecutionErrorDestinations.ts

```diff
@@ -0,0 +1,46 @@
+import { GetValueCommand } from "@stedi/sdk-client-stash";
+import { stashClient } from "./clients/stash.js";
+import { PARTNERS_KEYSPACE_NAME } from "./constants.js";
+import { ErrorWithContext } from "./errorWithContext.js";
+import {
+  DestinationErrorEvents,
+  DestinationErrorEventsSchema,
+  destinationExecutionErrorKey,
+} from "./types/Destination.js";
+
+const stash = stashClient();
+
+export const loadExecutionErrorDestinations = async (): Promise<
```

Comment:
nit: could DRY this up a bit and just have a generic `loadErrorDestinations` module that includes both `loadExecutionErrorDestinations` and `loadFileErrorDestinations` functions which could call a shared implementation with the appropriate stash key as input.

Reply:
both `loadExecutionErrorDestinations` and `loadFileErrorDestinations` return `Promise<DestinationErrorEvents>`, and it looks like the only difference in their implementation is the stash key used for the lookup, which is why I made the suggestion. nbd to keep them separate though